### PR TITLE
feat: [BD-46] implement rtl for components from Figure to Input Group

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Implement RTL support for all component
- from Figure to Input Group
- JIRA: https://openedx.atlassian.net/browse/PAR-646

### CODE REVIEW

File: src/Form/README.md [hunk 1]
```diff
@@ -105,10 +105,11 @@ notes: |
   <Form.Group id="formGridCheckbox">
     <Form.Check type="checkbox" id="formGridCheckboxCheck" label="Check me out" />
   </Form.Group>
-
+  <div className="d-flex">
   <Button variant="primary" type="submit">
     Submit
   </Button>
+  </div>
```
Comment (adamstankiewicz) [inline src/Form/README.md:112]: nit: Should lines 109-111 be indented?

File: src/Form/README.md [hunk 2]
```diff
@@ -181,9 +182,10 @@ notes: |
   <Form.Group id="formGridCheckbox-2">
     <Form.Check type="checkbox" id="formGridCheckboxCheck" label="Check me out" />
   </Form.Group>
-
+  <div className="d-flex">
   <Button variant="primary" type="submit">
     Submit
   </Button>
```
Comment (adamstankiewicz) [inline src/Form/README.md:188]: nit: Should lines 186-188 be indented?

File: src/Form/form-checkbox.mdx [hunk 3]
```diff
@@ -127,13 +127,15 @@ underlying `input` node. See MDN for documentation on available
...
   return (
     <>
-      <Form.Checkbox
-        checked={allChecked}
-        isIndeterminate={isIndeterminate}
-        onChange={handleCheckAllChange}
-      >
-        All the cheese
-      </Form.Checkbox>
+      <div className="d-flex">
```
[the remark this answers is not in the document]
Comment (vlasovmichael) [reply src/Form/form-checkbox.mdx:130]: We can avoid it. `d-flex` needed only to move the button to the right side in RTL direction. The screen without `d-flex` 
image

File: src/Form/README.md (hunk 2 above)
Comment (vlasovmichael) [reply src/Form/README.md]: Of course. My fault. Fixed

File: src/Form/_Form.scss [hunk 4]
```diff
@@ -21,6 +21,19 @@ $select-icon-padding: .5625rem !default;
 
 .pgn__form-group {
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/Form/_Form.scss:22]: @vlasovmichael Got it, thanks for the clarification.

**Review by adamstankiewicz — APPROVED** [review]

**Comment by edx-webhook** [comment]
@vlasovmichael 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1084&entry.1484655318=https://github.com/vlasovmichael&entry.752974735=2022-02-11+11:20&entry.1917517419=2022-02-25+12:06&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**